src/iOS/Xamarin.iOS/Samples/Layers/DisplayScene/DisplayScene.cs

```diff
@@ -38,64 +35,61 @@ public override void ViewDidLoad()
         {
             base.ViewDidLoad();
 
-            // Execute initialization 
             CreateLayout();
             Initialize();
         }
 
         public override void ViewDidLayoutSubviews()
         {
-            // Setup the visual frame for the SceneView
-            _mySceneView.Frame = new CoreGraphics.CGRect(0, 0, View.Bounds.Width, View.Bounds.Height);
-
-            base.ViewDidLayoutSubviews();
+            try
+            {
+                nfloat topMargin = NavigationController.NavigationBar.Frame.Height + UIApplication.SharedApplication.StatusBarFrame.Height;
+
+                // Reposition controls.
+                _mySceneView.Frame = new CoreGraphics.CGRect(0, 0, View.Bounds.Width, View.Bounds.Height);
+                _mySceneView.ViewInsets = new UIEdgeInsets(topMargin, 0, 0, 0);
+
+                base.ViewDidLayoutSubviews();
+            }
+            catch (NullReferenceException)
+            {
+            }
         }
 
         private void Initialize()
         {
-            // Create a new scene
-            Scene myScene = new Scene();
-
-            // Crate a new base map using the static/shared create imagery method
-            Basemap myBaseMap = Basemap.CreateImagery();
-
-            // Add the imagery basemap to the scene's base map property
-            myScene.Basemap = myBaseMap;
-
-            // Add scene (with an imagery basemap) to the scene view's scene property
-            _mySceneView.Scene = myScene;
-
-            // Create a new surface
-            Surface mySurface = new Surface();
-
-            // Define the string that points to the elevation image service
-            string myElevationImageService = "http://elevation3d.arcgis.com/arcgis/rest/services/WorldElevation3D/Terrain3D/ImageServer";
+            // Create and show a new scene with imagery basemap.
+            _mySceneView.Scene = new Scene(Basemap.CreateImagery());
```

Comment:
Since this sample shows a _very_ basic use-case, the audience is total Runtime notives. Because of that, I actually liked having the steps spelled out more.